frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -280,11 +285,36 @@ export const ClusterServiceVersionsDetailsPage: React.StatelessComponent<Cluster
       .concat(providedAPIsFor(obj).map((desc: CRDDescription) => ({
         href: referenceForProvidedAPI(desc),
         name: desc.displayName,
-        /* eslint-disable-next-line react/display-name */
         component: (instancesProps) => <ProvidedAPIPage {...instancesProps} csv={obj} kind={referenceForProvidedAPI(desc)} namespace={props.match.params.ns} />,
       })));
   };
 
+  const subscriptionStateToProps = ({k8s}: RootState, {installedCSV}: Pick<SubscriptionViewProps, 'installedCSV'>) => {
```

Comment:
Grab the `Subscription` and `PackageManifest` from Redux instead of adding another `<Firehose>`.

Reply:
Can you elaborate on why we want to do this instead of using `Firehose`?

Are we certain the data will be in Redux?

Reply:
We knew they would be in Redux because they are added as part of `props.resources` to `DetailsPage`. However, changed to `Firehose` for other reasons.